tests/neg-macros/quote-type-variable-no-inference-2.scala

```diff
@@ -0,0 +1,8 @@
+import scala.quoted.*
+
+def test2(x: Expr[Any])(using Quotes) =
+  x match
+    case '{ $_ : F[t, t]; () } => // warn // error
+    case '{ type u <: Int & Double; $_ : F[u, u] } =>
+
+type F[x <: Int, y <: Double]
```

Comment:
Could you add an example involving an F-bounds (like `type Foo[T <: Comparable[T]]`)?

Reply:
Added am example in `tests/neg-macros/quote-type-variable-no-inference-3.scala`